components/crypto/build.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+
+use exonum_build::ProtobufGenerator;
+
+fn main() {
+    gen_proto_files();
```

Comment:
If you add `#[cfg(feature = "with-protobuf")]` above the `gen_proto_files();` then you can get rid of 35 and 36 lines.